todo.js

```diff
@@ -15,3 +15,54 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+let process = require('process');
+let command = process.argv[2];
+let secondCommand = process.argv[3];
+// takes the third argument because the first two are the node and the location of the file
+let fs = require('fs');
+
+// file system, reads files
+
+function openAndRead(filename) {
```

Comment:
@chloedb See my comments on @wismith's code:

- https://github.com/wismith/project-js-todo-i/pull/1
- https://github.com/wismith/project-js-todo-i/pull/4

Rather than seeing your job as "reading from and writing to the `todos.txt` file", think of your job as modeling — in code — the behavior of a TODO list. The fact that we read the todo list in from a file, print it to a console, and (later) write to a file are things the core logic doesn't need to concern itself with.

For example, have functions that:

- Return a todo list given a file name
- Modify the todo list
- Display the todo list
- Write the contents of the todo list to a file
- etc.

Each function does one and only one thing. Everything each function needs to do its work arrives via arguments.

Imagine trying to sanity check (test) the `openAndRead` function. How would you do it?! The fact that it's really hard to do is a sign that the function is doing too much.

Right now, every part of your code has to know about every other part of your code to do its job. This means if any one part changes, you will have to check every other part of your code to verify that the overall program still runs as expected.